contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java

```diff
@@ -299,18 +312,15 @@ private File fetchFromSource(String sourceFileUrl,
                 logger.error("Source " + path.toString() + " should be a directory");
                 return null;
             }
-        } catch(Exception e) {
+        } catch (Exception e) {
             if(stats != null) {
                 stats.reportError("File fetcher failed for destination " + destinationFile, e);
             }
-            String errorMessage = "Error thrown while trying to get data from Hadoop filesystem : ";
-            logger.error(errorMessage, e);
```

Comment:
Can the error be still logged ? May be without the call stack or is it logged some where else ?